### ISSUE TITLE
[ProgressBar] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1271]
### Feedback

❌ 2 of the loaders on the doc page have no label specified.
❌ No label should be an error - or allow aria-label to be specified directly
❌ Indeterminate read as 50% (need to file w/ Apple)

### Path forward

1. Label the loaders!
2. `aria-label` is allowed, clarify system wide API surface at the intersection of visible labels and `aria-label` activation
3. James, can you share a bug report to track on this?

cc @jnurthen

**Comment by jnurthen** [comment]
I had previously filed this https://bugs.webkit.org/show_bug.cgi?id=214330
